cookbooks/aws-parallelcluster-environment/spec/unit/recipes/config_default_user_home_spec.rb

```diff
@@ -22,7 +22,31 @@
           expect(chef_run.node['cluster']['cluster_user_home']).to eq('/local/home/user')
           is_expected.to start_service("sshd")
         end
+
+        it 'moves the cluster user home directory with data integrity check' do
+          user_home = "/home/user"
+          user_local_home = "/local/home/user"
+          cluster_user = chef_run.node['cluster']['cluster_user']
+          expect(chef_run).to run_bash("Move #{user_home}").with(
+            code: <<-CODE
+    set -e
```

Comment:
Minor: is this indentation required? wCan't we place the code with the indentation aligned to `code:`?

Reply:
Unfortunately it can't. Or the test will fail. But we can do that use Regular Expression. But it need time and not necessary, and will make the code hard to read.

Reply:
np, let's leave it as it is.